assets/scss/core/_typography.scss

```diff
@@ -25,6 +25,7 @@ $dp-headers-h4-font-size: 22px;
 $dp-headers-h5-font-size: 18px;
 $dp-headers-h6-font-size: 13px;
 $dp-font-size-small: 13px;
```

Comment:
sorry, but it seems that there are some duplicated font-sizes, can you refactor the variables in order to only have once times declare the 13 size? (not sure if 14px it's repeated or not)